src/lib/webpack-config.js

```diff
@@ -201,6 +201,36 @@ export default env => {
 			}
 		}),
 
+		customConfig({
+			module: {
+				loaders: [
+					{
+						enforce: 'pre',
+						test: /\.tsx?$/,
+						use: [
+							{
+								loader: resolve(__dirname, './npm-install-loader'),
+								options: {
+									modules: ['typescript', 'awesome-typescript-loader'],
+									save: true
+								}
+							},
+							{
+								loader: 'awesome-typescript-loader',
+								options: {
+									sourceMap: true,
+									useBabel: true,
+									babelOptions: createBabelConfig(env),
+									useCache: true,
+									configFileName: resolve(__dirname, 'ts-config.js'),
```

Comment:
`tsconfig.json` is the standard config file name.  Is there any particular reason for the requiring a non-standard config file?